redbot/core/utils/chat_formatting.py

```diff
@@ -278,7 +282,7 @@ def pagify(
             yield in_text
 
 
-def strikethrough(text: str) -> str:
+def strikethrough(text: str, escape_formatting: bool = True) -> str:
     """Get the given text with a strikethrough.
 
     Note: This escapes text prior to applying a strikethrough
```

Comment:
This needs an update, maybe:
```suggestion
    Note: By default, this function will escape ``text`` prior to applying a strikethrough.
```